test/hexer_package_SUITE.erl

```diff
@@ -83,6 +83,18 @@ publish(_Config) ->
        catch _:{hexer_package, has_contributors} -> ok
        end,
 
+  ct:comment("Error: Bag Github tag"),
+  AppSrcBin3 = <<"{ application, hexer "
+                 "  , [ {vsn, git} ]"
+                 "}.">>,
+  ok = file:write_file("src/hexer.app.src", AppSrcBin3),
+  OsFalseFun = fun(_) ->  "fatal: " ++ (ErrorTag = "No Tag!!") end,
```

Comment:
If you define `ErrorTag` in the body of a fun, it will not be bound outside of it. Just use the same string in both places, not a variable.